lib/bitshares.js

```diff
@@ -159,6 +159,24 @@ export default class BitShares {
     )
   }
 
+    static async getLimitOrders(quoteSymbol, baseSymbol, limit = 50) {
+    if (limit > 100) limit = 100;
+        return BitShares.db.get_limit_orders(
```

Comment:
This static function, use `this` without `BitShares`.
Generally, I want to rewrite this (and other) class for use arrow functions in the future.

Reply:
ok, thx